consensus/hotstuff_mempool.go

```diff
@@ -0,0 +1,141 @@
+package consensus
+
+import (
+	"fmt"
+	"log"
+	"math/rand"
+	"sync"
+	"time"
+
+	"github.com/golang/protobuf/proto"
+	typesCons "github.com/pokt-network/pocket/consensus/types"
+	"github.com/pokt-network/pocket/shared/codec"
+	"github.com/pokt-network/pocket/shared/crypto"
+	mempool "github.com/pokt-network/pocket/shared/mempool/list"
+)
+
+// TODO: index HighQC incrementally
+// TODO: expose findHighQC
+// TODO: implement iterator
+// TODO: tests for list as well
```

Comment:
Can you just add a bit more details here. I realize it was a note you made to yourself at some point, but pretty confusing to the reader.

Reply:
💯 these are going to disappear soon